docs/data/date-pickers/localization/data.json

```diff
@@ -1,4 +1,12 @@
 [
+  {
+    "languageTag": "ru-RU",
```

Comment:
I believe it should be alphabetically ordered (based on `localeName`).
Did you run `yarn l10n` to generate the result or added it manually?

Reply:
manually

Reply:
Your latest pipeline before the recent push complained about just this fact [and "asked" if you forgot to run `yarn l10n` and commit changes](https://app.circleci.com/pipelines/github/mui/mui-x/32620/workflows/216fc1e6-e932-4367-9ba3-2da4d1305f09/jobs/187109)? 😃